tools/publish-lib.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Create one JS library per tag and commit the result to the gh-pages branch
+ * 
+ * node tools/publish-tags.js
+ */
+
+const util = require('util');
+const fs = require('fs');
+const execFile = util.promisify(require('child_process').execFile);
+
+async function main() {
+  console.log('Retrieve list of tags...');
+  const { stdout: tagOut } = await execFile('git', ['tag', '--sort=-v:refname']);
+  const tags = tagOut.split(/[\r\n]/).filter(tag => !!tag);
```

Comment:
should we also filter tags to keep only those that match the semver pattern?

Reply:
Latest code does that, more because it was more convenient to filter tags based on that than anything else. It could perhaps be useful to release other types of tags in the future, but we don't need that for now.